spec/unit/installer/xcode/pods_project_generator/pod_target_integrator_spec.rb

```diff
@@ -85,13 +86,7 @@ class PodTargetIntegrator
               end
 
               it 'excludes test framework and resource paths from dependent targets' do
-                test_framework_paths = [{ :name => 'TestVendored.framework',
```

Comment:
this test was a false positive. Now its more real by ensuring that a resource bundle of another test spec is not part of the input/output paths of another test native target.

Reply:
It was a false positive because the `banana_pod_target` never had test specs so it was never loading or returning paths for a test spec.

```ruby
-                @banana_pod_target.stubs(:resource_paths).returns('BananaLib' => [], 'BananaLib/Tests' => test_resource_paths)
```

this was stupid.